.github/workflows/CI.yml

```diff
@@ -0,0 +1,86 @@
+name: CI
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.14
```

Comment:
should we pin a go version here, or let setup-go install the latest go so that we don't have to manually update this for each go release?

if we decide to pin versions can we use latest 1.16?

Reply:
Yes, it could be latest, I have pinned only for consistency with CircleCI https://github.com/pantheon-systems/autotag/blob/master/.circleci/config.yml#L7

Reply:
It even could be a matrix with few versions(https://github.com/pantheon-systems/autotag/pull/54/commits/04f67be25fe2b7560f11b6508a0b036142e67022), but I reverted that.

Reply:
ah i see. We normally use our internal go-build container which always tracks with latest release. We didn't do that here because we were using circle's public image. I think it would save us time to not pin go versions. In the unlikely case that autotag becomes incompatible with a go 1.x release we can pin until we have time to fix any issues.

Reply:
@vtopc If this provided a library for others I would want to do a test matrix of go versions. Because the only supported interface for users to autotag is by using the binary I don't think we need a test matrix.

Reply:
Hm, interesting. It's using `go version go1.15.11 linux/amd64` by default as for now. Is it OK? Or should I pin to 1.16?

Reply:
@vtopc i noticed that in a personal project last weekend too and couldn't figure out why. I think it's OK, and I assume they'll move to 1.16 soon.